{"uid":"36b63bc64bf7f4f6","name":"test_query_unsupported_network[3-3]","fullName":"tests.agent_trajectory.feed_explore#test_query_unsupported_network","historyId":"d0aa82a76381055cce1d53079f03a727","time":{"start":1730345823196,"stop":1730345825508,"duration":2312},"status":"failed","statusMessage":"AssertionError: assert 'Unsupported network' in 'No activities found for the given address.'","statusTrace":"feed_explorer_agent = AgentExecutor(verbose=True, agent=RunnableMultiActionAgent(runnable=RunnableAssign(mapper={\n  agent_scratchpad: Runnab...ys_arg=[], stream_runnable=True), tools=[FeedExecutor(), FeedSourceExecutor(), DeFiExecutor(), TelegramNewsExecutor()])\n\n    @pytest.mark.asyncio\n    async def test_query_unsupported_network(feed_explorer_agent):\n        events = feed_explorer_agent.astream_events(\n            {\"messages\": [HumanMessage(content=\"Show me activities on the XYZ network\", name=\"human\")]}, version=\"v1\"\n        )\n    \n        tool_end_count = 0\n        async for event in events:\n            if event[\"event\"] == \"on_tool_end\":\n                tool_end_count += 1\n                assert \"output\" in event[\"data\"]\n>               assert \"Unsupported network\" in event[\"data\"][\"output\"]\nE               AssertionError: assert 'Unsupported network' in 'No activities found for the given address.'\n\nagent_trajectory/feed_explore.py:89: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"feed_explorer_agent","time":{"start":1730345815040,"stop":1730345823194,"duration":8154},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop_policy","time":{"start":1730345815040,"stop":1730345815040,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1730345823194,"stop":1730345823195,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: assert 'Unsupported network' in 'No activities found for the given address.'","statusTrace":"feed_explorer_agent = AgentExecutor(verbose=True, agent=RunnableMultiActionAgent(runnable=RunnableAssign(mapper={\n  agent_scratchpad: Runnab...ys_arg=[], stream_runnable=True), tools=[FeedExecutor(), FeedSourceExecutor(), DeFiExecutor(), TelegramNewsExecutor()])\n\n    @pytest.mark.asyncio\n    async def test_query_unsupported_network(feed_explorer_agent):\n        events = feed_explorer_agent.astream_events(\n            {\"messages\": [HumanMessage(content=\"Show me activities on the XYZ network\", name=\"human\")]}, version=\"v1\"\n        )\n    \n        tool_end_count = 0\n        async for event in events:\n            if event[\"event\"] == \"on_tool_end\":\n                tool_end_count += 1\n                assert \"output\" in event[\"data\"]\n>               assert \"Unsupported network\" in event[\"data\"][\"output\"]\nE               AssertionError: assert 'Unsupported network' in 'No activities found for the given address.'\n\nagent_trajectory/feed_explore.py:89: AssertionError","steps":[],"attachments":[{"uid":"5bacfb248c3e6062","name":"stdout","source":"5bacfb248c3e6062.txt","type":"text/plain","size":184},{"uid":"d4037b2609277f0c","name":"stderr","source":"d4037b2609277f0c.txt","type":"text/plain","size":294}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"event_loop::3","time":{"start":1730345825521,"stop":1730345825521,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1730345825521,"stop":1730345825521,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_restore_policy","time":{"start":1730345825521,"stop":1730345825521,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1730345825521,"stop":1730345825521,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.agent_trajectory"},{"name":"suite","value":"feed_explore"},{"name":"host","value":"fv-az1206-377"},{"name":"thread","value":"3630-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.agent_trajectory.feed_explore"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"__pytest_repeat_step_number","value":"2"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["asyncio"]},"source":"36b63bc64bf7f4f6.json","parameterValues":["2"]}